exporter/opentelemetry-exporter-richconsole/src/opentelemetry/exporter/richconsole/__init__.py

```diff
@@ -0,0 +1,174 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+The **OpenTelemetry Rich Console Exporter** provides a span exporter from a batch span processor
+to print `OpenTelemetry`_ traces using `Rich`_.
+
+Installation
+------------
+
+::
+
+    pip install opentelemetry-exporter-richconsole
+
+
+Usage
+-----
+
+The Rich Console Exporter is a console exporter that prints a tree view onto stdout of the traces
+with the related spans and properties as children of that tree. For the tree view, the Rich
+Console Exporter must be used with a BatchSpanProcessor. If used within a SimpleSpanProcessor,
```

Comment:
Just confirming that using it with a different processor will still work. If that's the case, maybe `must` can be changed to `should`